source/codegen/metadata/nixnet/functions.py

```diff
@@ -0,0 +1,1539 @@
+                'direction': 'in',
+                'name': 'propertySize',
+                'type': 'u32'
+            },
+            {
+                'direction': 'out',
+                'name': 'propertyValue',
+                'type': 'void[]'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DbUndeploy': {
+        'cname': 'nxdbUndeploy',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'ipAddress',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'databaseAlias',
+                'type': 'const char[]'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DisconnectTerminals': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'sessionRef',
+                'type': 'nxSessionRef_t'
+            },
+            {
+                'direction': 'in',
+                'enum': 'TerminalName',
+                'name': 'source',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'enum': 'TerminalName',
+                'name': 'destination',
+                'type': 'const char[]'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'Flush': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'sessionRef',
+                'type': 'nxSessionRef_t'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'FutureTimeTrigger': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'sessionRef',
+                'type': 'nxSessionRef_t'
+            },
+            {
+                'direction': 'in',
+                'name': 'when',
+                'type': 'nxTimestamp1ns_t'
+            },
+            {
+                'direction': 'in',
+                'enum': 'TimeScale',
+                'name': 'timescale',
+                'type': 'u32'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'GetProperty': {
+        'codegen_method': 'no',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'sessionRef',
+                'type': 'nxSessionRef_t'
+            },
+            {
+                'direction': 'in',
+                'enum': 'Properties',
+                'name': 'propertyID',
+                'type': 'u32'
+            },
+            {
+                'direction': 'in',
+                'name': 'propertySize',
+                'type': 'u32'
+            },
+            {
+                'direction': 'out',
+                'enum': 'PropertyValue',
+                'name': 'propertyValue',
+                'type': 'void[]'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'GetPropertySize': {
```

Comment:
We discussed using this as a size mechanism for `GetProperty` so that the user would not have to call it directly or pass it in. Is that something you're planning to do/tracking?

It's fine as-is for now. I think I even recommended that you add them like this first. So that's great! Just checking on what you're thinking.